.gitignore

```diff
@@ -13,3 +13,4 @@
 /virtcontainers/shim/mock/kata-shim/kata-shim
 /virtcontainers/utils/supportfiles
 /virtcontainers/profile.cov
+.idea/
```

Comment:
this shouldn't be included?

Reply:
Hah， it's `goland IDE`'s trick~

Reply:
This should go in its own separate commit.

Reply:
Yes please make sure we don't introduce this into the PR.